### PR TITLE
Sample coredns config and doc updates

### PR DESCRIPTION
**Pull request by taylor**
## Description
Example test suite configuration and docs

Issues:
- N/A

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [x] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes
* [ ]  Bug fix (non-breaking change which fixes an issue)
* [x]  New feature (non-breaking change which adds functionality)
* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
## Checklist:
* [ ]  My change requires a change to the documentation.
* [x]  I have updated the documentation accordingly.
* [ ] No updates required.

### CODE REVIEW

File: sample-cnfs/sample-coredns-cnf/README.md [hunk 1]
```diff
@@ -23,17 +23,25 @@ helmenv versions
...
 ```
-helm pull stable/coredns
+cd sample-cnfs/sample-coredns-cnf
+helm pull --untar stable/coredns
+cd ../..
+cp -a sample-cnfs/sample-coredns-cnf cnfs
 ```
-### Example cnf-conformance config file for sample-core-dns-cnf
-In ./cnfs/sample-core-dns-cnf/cnf-conformance.yml
+### Create confromance config.yml based on cnf-conformance.html
```
Comment (wavell) [inline sample-cnfs/sample-coredns-cnf/README.md]: cnf-conformance.html ?  I think you mean .yml

File: USAGE.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 The CNF Conformance Test suite can be run in developer mode (using crystal lang directly) or in production mode (using an executable).  See the [pseudo code documentation](https://github.com/cncf/cnf-conformance/blob/master/PSEUDO-CODE.md) for examples of how the internals of WIP tests might work.
 
+
+### Configuration
+
+Create a config.yml and configure helm directory, deployment name, etc
+
+See [config.yml.example](config.yml.example) for an example configuration.
```
Comment (wavell) [inline USAGE.md]: I think this needs to be cnf-conformance.yml.example

File: USAGE.md [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
+
+quick start: `cp config.yml.example config.yml`
```
Comment (wavell) [inline USAGE.md]: I think this needs to be cnf-conformance.yml.example

**Comment by taylor** [comment]
deprecated.  closing

**Pull request closed without merging**